index.html

```diff
@@ -2,14 +2,7 @@
 <html lang="en">
 
 <head>
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-116597696-1"></script>
-    <script>
-        window.dataLayer = window.dataLayer || [];
-        function gtag() { dataLayer.push(arguments); }
-        gtag('js', new Date());
-        gtag('config', 'UA-116597696-1');
-    </script>
+   
```

Comment:
If you could remove this blank line, this is good to go :) 
Thanks for your contribution.

Reply:
okay